modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_tag_selector.scss

```diff
@@ -1,83 +0,0 @@
-.lfr-tags-selector-list {
-	border: 0 solid transparent;
-	border-width: 0 1px 1px;
-	margin-bottom: 0.5em;
-
-	li {
-		&.tree-node .tree-node-content {
-			@include text-truncate();
-		}
-	}
-}
-
-.lfr-tags-selector-content {
-	&.has-error .form-feedback-item {
-		margin-top: -1.5em;
-	}
-
-	.toolbar {
-		position: static;
-	}
-
-	.textboxlistentry-holder li {
-		margin: 0 5px 0 0;
-	}
-
-	.textboxlist-input-container {
-		input {
-			margin: 0;
-		}
-
-		.textboxlist-content .field-input-text {
-			padding: 0;
-		}
-	}
-}
```

Comment:
This is still used in several places

Reply:
after my review I think we can remove them, yeah!